pkg/sync/v14/sync.go

```diff
@@ -280,6 +280,12 @@ var (
 	vwcFilter = func(o unstructured.Unstructured) bool {
 		return o.GroupVersionKind().GroupKind() == schema.GroupKind{Group: "admissionregistration.k8s.io", Kind: "ValidatingWebhookConfiguration"}
 	}
+	roleFilter = func(o unstructured.Unstructured) bool {
+		return o.GroupVersionKind().Kind == "ClusterRole" || o.GroupVersionKind().Kind == "Role"
+	}
+	roleBindingFilter = func(o unstructured.Unstructured) bool {
+		return o.GroupVersionKind().Kind == "ClusterRoleBinding" || o.GroupVersionKind().Kind == "RoleBinding"
+	}
```

Comment:
without setting the order, metrics-server deployment is created before the role bindings and it will be in CrashLoopBackoff for a short time.  But the deployment is functional shortly after that, so these changes are not crucial